babel.config.js

```diff
@@ -1 +1,10 @@
-module.exports = require('./shell/babel.config.js');
```

Comment:
Careful with this change, since in the shell we have some configuration changed, including the one where we allow the use of `require`, otherwise it creates error:

```
TypeError: require.context is not a function
```